core/src/apps/nem/transfer/__init__.py

```diff
@@ -12,11 +12,12 @@ async def transfer(
     common: NEMTransactionCommon,
     transfer: NEMTransfer,
     node: bip32.HDNode,
+    chunkify: bool = False,
```

Comment:
argument here and in layout.ask_transfer does not need a default value

Reply:
Done in [7e03759](https://github.com/trezor/trezor-firmware/pull/3237/commits/7e03759ea7c3f0e2e56218ee1bd5cb18bef9c134)